Integrations/integration-mcafeeDam.yml

```diff
@@ -199,28 +272,76 @@ script:
         case 'fetch-incidents':
             return JSON.stringify(fetchIncidents());
         case 'dam-get-alert-by-id':
-            return sendRequest(finalUrl, 'alert', 'HH$Id=' + encodeURIComponent(args.id)).HedgehogXmlApi.alert;
+            var jsonRes=sendRequest(finalUrl, 'alert', 'HH$Id=' + encodeURIComponent(args.id)).HedgehogXmlApi.alert;
+            if (jsonRes) {
+                var entryContext = {
+                };
+                md = alertToMd(jsonRes);
+                return {Type: entryTypes.note, Contents: jsonRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown, EntryContext: {}};
+            } else {
+                md = '### no alerts found\n';
+                return {Type: entryTypes.note, Contents: null, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown};
+            }
         case 'dam-get-latest-by-rule':
             var count = args.count || 10;
             var timeBack = 1000 * 60 * (args.timeBack || 10);
             var res = sendRequest(finalUrl, 'alert', 'HH$RuleName=' + encodeURIComponent(args.ruleName) +
                 '&HH$TimeBackPeriod='+encodeURIComponent(timeBack +'') + '&HH$pageSize='+encodeURIComponent(count)).HedgehogXmlApi;
+
             if (res && res.alert) {
-                return {alerts: res.alert};
+                jsonRes = res.alert;
+                console.log(JSON.stringify(jsonRes));
+                var entryContext = {};
+                md = alertsToMd(jsonRes);
+                return {Type: entryTypes.note, Contents: jsonRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown, EntryContext: {}};
+            } else {
+                md = '### no alerts found\n';
+                return {Type: entryTypes.note, Contents: null, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown};
             }
-            return null;
     }
   type: javascript
   commands:
   - name: dam-get-alert-by-id
+    deprecated: false
     arguments:
     - name: id
       required: true
       default: true
       description: The alert ID
+    outputs:
+    - contentPath: id
+      contextPath: AlertId
+      description: DAM Alert ID
+    - contentPath: accessedObjects.object.name
+      contextPath: alertAccessedObjects
+      description: DAM Accessed Objects
+      important: true
+      importantDescription: DAM Accessed Objects
+    - contentPath: execUser
+      contextPath: dbUser
```

Comment:
Wonder if these are the right paths to export. Should we do `dam.dbUser`, `dam.osUser`, etc.?